config/m4/mpi.m4

```diff
@@ -26,6 +26,8 @@ AS_IF([test "x$with_mpi" = xyes],
         [
         AC_ARG_VAR(MPICC,[MPI C compiler command])
         AC_PATH_PROGS(MPICC,mpicc mpiicc,"",$mpi_path)
+        AC_ARG_VAR(SHMEMCC,[SHMEM C compiler command])
+        AC_PATH_PROGS(SHMEMCC,shmemcc,"",$mpi_path)
```

Comment:
isn`t it ```oshcc```?

Reply:
AFAIK `shmemcc` is the standard name

Reply:
shmemcc, this goes back to SGI implementation